src/liblsquic/lsquic_send_ctl.c

```diff
@@ -924,7 +926,7 @@ static void
 send_ctl_expire (lsquic_send_ctl_t *ctl, enum expire_filter filter)
 {
     lsquic_packet_out_t *packet_out, *next;
-    int n_resubmitted;
+    int n_resubmitted =0;
```

Comment:
This is uninitialized when `filter` is not one of the enum values, which is never the case.